tools/rosbag_storage/CMakeLists.txt

```diff
@@ -50,3 +56,17 @@ install(DIRECTORY include/
   DESTINATION ${CATKIN_GLOBAL_INCLUDE_DESTINATION}
   FILES_MATCHING PATTERN "*.h"
 )
+
+if(NOT WIN32)
+  install(FILES encryptor_plugins.xml
+    DESTINATION ${CATKIN_PACKAGE_SHARE_DESTINATION}
+  )
+
+  if(CATKIN_ENABLE_TESTING)
+    find_package(rostest)
+
+    catkin_add_gtest(test_aes_encryptor test/test_aes_encryptor.cpp
+      WORKING_DIRECTORY ${PROJECT_SOURCE_DIR}/test)
+    target_link_libraries(test_aes_encryptor rosbag_storage ${catkin_LIBRARIES} ${Boost_LIBRARIES})
```

Comment:
This line needs to be wrapped in a conditional: `if(TARGET test_aes_encryptor)`.

Reply:
Addressed.  Thank you.